GANDLF/entrypoints/update_version.py

```diff
@@ -46,20 +46,20 @@ def in_place_string_replace(
 
     cwd = os.getcwd()
     in_place_string_replace(
-        os.path.join(cwd, "GANDLF/version.py"),
+        os.path.join(cwd, "../version.py"),
```

Comment:
That's a weird refactoring with paths in this file, I didn't expected this. Treat this file as broken for now.

Reply:
Resolved: update_version is moved back from cli tools to usual python script in the root directory.